src/Kunstmaan/NodeBundle/Tests/Entity/TestEntity.php

```diff
@@ -87,4 +89,17 @@ public function isStructureNode()
     {
         return false;
     }
+
+	/**
+	 * @return PageTab[]
+	 */
+    public function getTabs()
+	{
+		return [
+			(new PageTab())
+				->setTabTitle('tab1_title')
+				->setInternalName('tab1_name')
+				->setPosition(1)
```

Comment:
There is no `formType` configured, shouldn’t it be required? see my other comment 👇